README.md

```diff
@@ -10,8 +10,15 @@
 [![License: MIT](https://img.shields.io/badge/License-MIT-success.svg)](https://opensource.org/licenses/MIT)
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.10797541.svg)](https://zenodo.org/doi/10.5281/zenodo.10797541)
 
-**TrixiParticles.jl** is a numerical simulation framework designed for particle-based numerical methods, with an emphasis on multiphysics applications, written in [Julia](https://julialang.org).
-A primary goal of the framework is to be user-friendly for engineering, science, and educational purposes. In addition to its extensible design and optimized implementation, we prioritize the user experience, including installation, pre- and postprocessing.
+**TrixiParticles.jl** is a high-performance numerical simulation framework for particle-based methods, focused on the simulation of complex multiphysics problems, and written in [Julia](https://julialang.org).
+
+TrixiParticles.jl focuses on the following use cases:
+- Development of new particle-based methods and models.
+- Accurate and efficient physics-based modelling of complex multiphysics problems.
+- Easy setup of accessible simulations for educational purposes, including student projects, coursework, and thesis work.
```

Comment:
```suggestion
TrixiParticles.jl primarily focuses on the following use cases:
- Accurate and efficient physics-based modelling of complex multiphysics problems.
- Development of new particle-based methods and models.
- Easy setup of accessible simulations for educational purposes, including student projects, coursework, and thesis work.
```
I would put multiphysics first (since it's part of what sets it apart from your standard SPH code)